src/compositemark/boxplot.ts

```diff
@@ -155,6 +146,16 @@ export function box2DOrient(spec: GenericUnitSpec<Encoding<Field>, BOXPLOT | Box
   const yEncChannel = encoding.y as FieldDef<Field>;
   let resultOrient: Orient;
 
+  if (isContinuous(encoding.x) && isContinuous(encoding.y)) {
+    const xEncChannel = encoding.x as FieldDef<Field>;
+    const yEncChannel = encoding.y as FieldDef<Field>;
+    if (xEncChannel.aggregate === BOXPLOT && yEncChannel.aggregate === BOXPLOT) {
+      throw new Error('Both x and y cannot have aggregate');
```

Comment:
I will need to add redundant logic into box2DOrient that's already done in box2DParams to find the continuous axis if I want to throw the error in box2DOrient